students/lauraannf/lessons/lesson01/assignment/test_unit.py

```diff
@@ -0,0 +1,146 @@
+                                             'material': 'material',
+                                             'size': 'size'}}
+            self.assertEqual(test_return_furniture, test_inventory)
+        with patch('builtins.input', side_effect=electric_input):
+            test_inventory = add_new_item()
+            test_return_electrics = {'100': {'product_code': '100',
+                                               'description': 'description',
+                                               'market_price': 24,
+                                               'rental_price': 'rentalprice',
+                                               'material': 'material',
+                                               'size': 'size'},
+                                       '200': {'product_code': '200',
+                                               'description': 'description',
+                                               'market_price': 24,
+                                               'rental_price': 'rentalprice',
+                                               'brand': 'brand',
+                                               'voltage': 'voltage'}}
+            self.assertEqual(test_return_electrics, test_inventory)
+        with patch('builtins.input', side_effect=other_input):
+            test_inventory = add_new_item()
+            test_return_other = {'100': {'product_code': '100',
+                                         'description': 'description',
+                                         'market_price': 24,
+                                         'rental_price': 'rentalprice',
+                                         'material': 'material',
+                                         'size': 'size'},
+                                 '200': {'product_code': '200',
+                                         'description': 'description',
+                                         'market_price': 24,
+                                         'rental_price': 'rentalprice',
+                                         'brand': 'brand',
+                                         'voltage': 'voltage'},
+                                 '300': {'product_code': '300',
+                                         'description': 'description',
+                                         'market_price': 24,
+                                         'rental_price': 'rentalprice'}}
+            self.assertEqual(test_return_other, test_inventory)
+
+    def test_item_info(self):
+        FULL_INV = {'100': {'product_code': '100',
+                            'description': 'description',
+                            'market_price': 24,
+                            'rental_price': 'rentalprice',
+                            'material': 'material',
+                            'size': 'size'}}
+        with patch('builtins.input', side_effect='100'):
+            test_info = item_info(FULL_INV)
```

Comment:
This one is creating a circular dependency, in that you're comparing your code's return with something generated by the code itself.